src/_product-updates/2025-03-05-badges.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Introducing state badges for operations and properties 
+tags: [New]
+image: /images/changelog/badges.png
+---
+
+![api-explorer-auth.png](/images/changelog/badges.png)
+
+Operations and properties go through multiple states during their lifetime, from `unstable` to `beta` or `release candidate`.  This information can be crucial for your users. That's why we are introducing a new property named `x-state`. 
```

Comment:
```suggestion
Operations and properties go through multiple states during their lifetime, from `unstable` to `beta` or `release candidate`.  This information can be crucial for your users. And more importantly, it can be completely specific to your API lifecycle. That's why we are introducing a new property named `x-state`. 
```